src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -295,7 +329,8 @@ func (b *ByocAws) WhoAmI(ctx context.Context) (*defangv1.WhoAmIResponse, error)
 }
 
 func (*ByocAws) GetVersions(context.Context) (*defangv1.Version, error) {
-	cdVersion := CdImage[strings.LastIndex(CdImage, ":")+1:]
+	cdVersion := getCdImage(byoc.CdLatestImageTag)
+	cdVersion = cdVersion[strings.LastIndex(cdVersion, ":")+1:]
 	return &defangv1.Version{Fabric: cdVersion}, nil
```

Comment:
I think this became much easier, since we have the separated tag now:
```suggestion
	return &defangv1.Version{Fabric: byoc.CdLatestImageTag}, nil
```